receiver/jaegerreceiver/jaeger_agent_test.go

```diff
@@ -50,10 +73,98 @@ func TestJaegerAgentUDP_ThriftBinary_6832(t *testing.T) {
 	})
 }
 
+func TestJaegerAgentUDP_ThriftBinary_PortInUse(t *testing.T) {
+	// This test confirms that the thrift binary port is opened correctly.  This is all we can test at the moment.  See above.
+	port := testutils.GetAvailablePort(t)
+
+	config := &Configuration{
+		AgentBinaryThriftPort: int(port),
+	}
+	jr, err := New(context.Background(), config, nil, zap.NewNop())
+	if err != nil {
```

Comment:
nit: use `require.NoError(t, err)` - there similar statements below, pointing only this one

Reply:
Removed all `err != nil` checks in favor of `require.NoError()`